cdist/conf/type/__daemontools/files/init.d-svscan

```diff
@@ -23,33 +23,33 @@ fi
 
 case "$1" in
     start)
-        echo -n "Starting daemontools: "
-        if [ ! `pidof svscan` ]; then
-                echo -n "svscan "
+        printf "Starting daemontools: "
+        if [ ! "$(pidof svscan)" ]; then
```

Comment:
This command substitution is not needed, the return code well defined (0 if at least one pgm found, 1 if no pgm found).

Reply:
I stand by this.